ic-auction/src/types.rs

```diff
@@ -0,0 +1,40 @@
+use ic_cdk::export::candid::{CandidType, Deserialize};
+use ic_helpers::tokens::Tokens128;
+
+pub type Cycles = u64;
+pub type TxId = u64;
+pub type Timestamp = u64;
+
+#[derive(CandidType, Debug, Clone, Deserialize, PartialEq)]
+pub struct AuctionInfo {
```

Comment:
Do we need this type to be public or can it be `pub(crate)` ?
Same for all the fields

Reply:
Yes, because we overwrite `disburse_rewards` as a part of public API